pr-commit-name-validator/action.yml

```diff
@@ -0,0 +1,35 @@
+name: 🔍️ PR Commit Name Validation
+author: "Erik Stokken Samuelsson"
```

Comment:
Maybe we should generally just have 'Farfetch'd Team' as author on all of these things? Non-blocking.